### PR TITLE
Adds 4 games

### PR DESCRIPTION
**Pull request by joshandersn**
black ops 3
human revolution
weird west
sleeping dogs

### CODE REVIEW

File: src/game-support/call-of-duty-bo-3.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Call of Duty: Black Ops III
+<!-- script:Aliases [] -->
```
Comment (mattmaniak) [inline src/game-support/call-of-duty-bo-3.md:2]: Add an alias of `Call of Duty: Black Ops III` please.

File: src/game-support/call-of-duty-bo-3.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+{{#template ../templates/rating.md status=Garbage installs=Yes opens=No}}
+
+Hoping to play online as macOS port is versions behind the windows port.
+
+> [!NOTE]
+> Black window upon startup.
+
```
Comment (mattmaniak) [inline src/game-support/call-of-duty-bo-3.md:10]: Remove unnecessary whitespace.

File: src/game-support/sleeping-dogs-de.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Sleeping Dogs: Definitive Edition
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Platinum installs=Yes opens=Yes}}
```
Comment (mattmaniak) [inline src/game-support/sleeping-dogs-de.md:4]: Downgrade to `Silver` due to the minor audio defect.

File: src/game-support/weird-west-de.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Weird West: Definitive Edition
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Platinum installs=Yes opens=Yes}}
```
Comment (mattmaniak) [inline src/game-support/weird-west-de.md:4]: Downgrade to `Bronze` due to the configuration need before gameplay and dense areas stuttering.

File: src/game-support/deus-ex-human-revolution.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Deus Ex: Human Revolution
+<!-- script:Aliases [] -->
```
Comment (mattmaniak) [inline src/game-support/deus-ex-human-revolution.md:2]: Add alias of `Deus Ex Human Revolution`.

File: src/game-support/sleeping-dogs-de.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# Sleeping Dogs: Definitive Edition
+<!-- script:Aliases [] -->
```
Comment (mattmaniak) [inline src/game-support/sleeping-dogs-de.md:2]: Add alias of `Sleeping Dogs Definitive Edition`.

File: src/game-support/weird-west-de.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# Weird West: Definitive Edition
+<!-- script:Aliases [] -->
```
Comment (mattmaniak) [inline src/game-support/weird-west-de.md:2]: Add alias of `Weird West Definitive Edition`.

**Review by mattmaniak — CHANGES_REQUESTED** [review]

File: src/game-support/sleeping-dogs-de.md (hunk 3 above)
Comment (joshandersn) [reply src/game-support/sleeping-dogs-de.md:4]: Found a fix for the audio. could we settle for Gold?

...pushing changes now

File: src/game-support/call-of-duty-bo-3.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+# Call of Duty: Black Ops III
+<!-- script:Aliases [
+    "Call of Duty: Black Ops III",
```
Comment (mattmaniak) [inline src/game-support/call-of-duty-bo-3.md:3]: Write this alias without a colon as it should differ from the game title.

File: src/game-support/deus-ex-human-revolution.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
+# Deus Ex: Human Revolution
+<!-- script:Aliases [
+    "Deus Ex: Human Revolution"
```
Comment (mattmaniak) [inline src/game-support/deus-ex-human-revolution.md:3]: Write this alias without a colon as it should differ from the game title.

File: src/game-support/sleeping-dogs-de.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
+# Sleeping Dogs: Definitive Edition
+<!-- script:Aliases [
+    "Sleeping Dogs: Definitive Edition"
```
Comment (mattmaniak) [inline src/game-support/sleeping-dogs-de.md:3]: Write this alias without a colon as it should differ from the game title.

File: src/game-support/weird-west-de.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
+# Weird West: Definitive Edition
+<!-- script:Aliases [
+    "Weird West: Definitive Edition"
```
Comment (mattmaniak) [inline src/game-support/weird-west-de.md:3]: Write this alias without a colon as it should differ from the game title.

**Review by mattmaniak — COMMENTED** [review]
Remove colons from aliases that look like the game title please.

File: src/game-support/weird-west-de.md (hunk 11 above)
Comment (joshandersn) [reply src/game-support/weird-west-de.md]: All done, and thank you!

**Review by mattmaniak — APPROVED** [review]

**Review by hahayupgit — APPROVED** [review]
